### ISSUE TITLE
[Bug]: Unexpected token 'export' error when running SvelteKit dev server (vite)

### ISSUE THREAD

**Issue by nstuyvesant** [carbon-design-system/carbon-charts#1354]
### Contact Details

nstuyvesant@gmail.com

### What happened?

Using @carbon/charts in my SvelteKit app. When I run the dev server, I get this error:
```
(node:64429) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
Unexpected token 'export'
/Users/nates/dev/shy-svelte/node_modules/@carbon/charts/interfaces/index.js:1
export * from './a11y';
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:168:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:197:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
```

It seems the issue is that the package.json for @carbon/charts@0.57.0  does not have`  "type": "module"` even though @carbon/charts-svelte@0.57.0 does.

Wondering if this was intentional for @carbon/charts due to some historical reason or perhaps just an oversight?

### Version

@carbon/charts@0.57.0
@carbon/charts-svelte@0.57.0
@sveltejs/kit@1.0.0-next.320

### Data & options used

If I import anything from @carbon/charts-svelte without even using the chart component, the error occurs when I `npm run dev`.

For example:
```typescript
  import { DonutChart, LineChart } from '@carbon/charts-svelte'
```

**Comment by theiliad** [comment]
Hi @nstuyvesant,
Just a heads up that the svelte wrappers are community-contributed.

Have you tried any of these examples https://github.com/metonym/carbon-charts-svelte-examples?

**Comment by nstuyvesant** [comment]
Thanks - I will try those out. I have Carbon Charts working in production. The problem is only when using the dev server.

**Comment by nstuyvesant** [comment]
Hi @metonym,

Lots of changes since I submitted this issue:

1. `@carbon/charts` moved from rollup to vite for its bundler and now optimizes its own dependencies
2. `@carbon/charts` now uses `html-to-image` (ESM) instead of an IIFE `dom-to-image` (created challenges from vite dev)
3.  All types and enums in the `@carbon/charts` package are now exported top-level so developers no longer need to worry about namespaces like `@carbon/charts/interfaces`, `@carbon/charts/models`. etc. The old approach using those namespaces will continue to work however.
4. `@carbon/charts-svelte` re-exports some of the most commonly-used types and enums so Svelte developers don't have to add `@carbon/charts` as a direct dependency and import the types and enums from there.

Didn't realize I left this one open - thanks for catching it.

Closing it now as resolved.
